protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -501,8 +504,26 @@ func (k Keeper) internalCanUpdateSubaccounts(
 			)
 		}
 
-		if exists && currentBlock-lastBlockNegativeTncSubaccountSeen <
-			types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS {
+		// Panic if the current block is less than the last block a chain outage was seen.
+		fmt.Println("BLOCK TIME KEEPER:", k.blocktimeKeeper)
+		downtimeInfo := k.blocktimeKeeper.GetDowntimeInfoFor(ctx, 5*time.Minute)
```

Comment:
Used 5 minutes here since that's the shortest chain outage duration. I can create a ticket to check for outages on shorter durations.